src/screens/chat-screen/components/message-components/CommandOptionsMenu.tsx

```diff
@@ -270,8 +275,8 @@ export const CommandOptionsMenu = () => {
   const { bottom } = useSafeAreaInsets();
   const isAndroid = Platform.OS === 'android';
   const containerHeight = isAndroid
-    ? 150 + (bottom === 0 ? 16 : bottom)
-    : 110 + (bottom === 0 ? 16 : bottom);
+    ? 190 + (bottom === 0 ? 16 : bottom)
+    : 150 + (bottom === 0 ? 16 : bottom);
```

Comment:
Is it possible to increase the height of the bottom sheet container a bit more?
<img src="https://github.com/user-attachments/assets/1782b84b-74ca-4561-962c-69a8c6cdbe58" width="300"> </img>